openterrain/__init__.py

```diff
@@ -242,7 +338,7 @@ def hillshade(elevation, azdeg=315, altdeg=45, vert_exag=1, dx=1, dy=1, fraction
     # Because most image and raster GIS data has the first row in the array
     # as the "top" of the image, dy is implicitly negative.  This is
     # consistent to what `imshow` assumes, as well.
-    dy = -dy
+    dy = abs(dy)
```

Comment:
This is the thing I want to clear up. It eventually works right, but the sequence of flipping is unclear.